algo-core/ortools_solver.py

```diff
@@ -101,10 +103,14 @@ def setup_dataframes():
         ],
     )
 
-    df_n_indices = pd.MultiIndex.from_product(
-        [[t for t in range(num_tracks)], [d for d in range(num_days)]],
-        names=("track", "day"),
-    )
+    i_tuple = []
+
+    for t, track in enumerate(tracks):
+        if 19 in range(track["start_hour"], track["end_hour"]):
```

Comment:
Why 19? 
Would be good to have a comment about this special number.

Reply:
true, good point